src/skmatter/clustering/_quick_shift.py

```diff
@@ -20,19 +20,19 @@ class QuickShift(BaseEstimator):
     Parameters
     ----------
     dist_cutoff_sq : float, default=None
-        The squared distance cutoff for searching for the next point. If `None`, the
-        Gabriel graph is used.
+        The squared distance cutoff for searching for the next point. If :obj:`None`,
+        the Gabriel graph is used.
     gabriel_shell : int, default=None
         The number of neighbor shell of gabriel graph. If None, the distance cutoff
         is used.
     scale : float, default=1.0
         Distance cutoff scaling factor used during the QS clustering. It will be squared
         since we are using the squared distance.
-    metric : str, default='periodic_euclidean'
+    metric : Callable, default=``pairwise_euclidean_distances``
         The metric to use. Currently only one.
```

Comment:
There seems to be some more docs missing. The sentence also is weird. Please also give the synopsis you expect for the Callable. I think you can also give this as a type hint.